packages/server/src/index.ts

```diff
@@ -15,7 +15,7 @@ await createCacheDirectory()
  * Tickers
  */
 cmcTicker.startTicker(ms('1h'), true)
-githubTicker.startTicker(ms('5s'), true)
+githubTicker.startTicker(ms('5m'), true)
```

Comment:
Haha, yes 5sec won't be good no, but every 1min should be fine? Can set to 1 if ya want.

Reply:
for now let's just leave it until front gets stable again

Reply:
I'll create an issue for it so we don't forget.